.mk/local.mk

```diff
@@ -0,0 +1,39 @@
+##@ Local (Kind)
+
+.PHONY: create-kind-cluster
+create-kind-cluster: $(KIND) ## Create kind cluster
+	-$(KIND) create cluster --config config/kubernetes/kind.config.yaml
+	kubectl cluster-info --context kind-kind
+
+.PHONY: delete-kind-cluster
+delete-kind-cluster: $(KIND) ## Delete kind cluster
+	$(KIND) delete cluster
+
+.PHONY: local-deploy
+local-deploy: create-kind-cluster deploy-all  ## Local deploy (kind, loki, grafana and example-cr)
+
+.PHONY: clean-leftovers
+clean-leftovers:
+	-PID=$$(pgrep --oldest --full "main.go"); pkill -P $$PID; pkill $$PID
+	-kubectl delete namespace network-observability
+
+.PHONY: local-redeploy
+local-redeploy: clean-leftovers undeploy-all deploy-all  ## Local re-deploy (loki, grafana and example-cr)
+
+.PHONY: local-undeploy
+local-undeploy: clean-leftovers undeploy-all delete-kind-cluster  ## Local cleanup
+
+local-run: create-kind-cluster local-redeploy ## run the operator locally
```

Comment:
FYI when running `make local-run`, it successfully deploys the stack but there's some errors about missing namespace `openshift-network-operator` => this is something I've fixed in my PR https://github.com/netobserv/network-observability-operator/pull/97